tests/tasks/setup/eks/awscli-role.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-role-create
+  namespace: tekton-pipelines 
+spec:
+  description: |
+    Creates Roles from CFN json stack.
+    This Task can be used to create IM Roles using the CFN json/yaml stack.
+  results:
+  - name: role-arn
+    description: Stores the role arn created by the task
+  params:
+  - name: stack-name
+    description: The name of the Role name you want to spin.
+  - name: role-cfn-url
+    description: The url of the CFN YAML/JSON to create IAM Roles 
+  - name: region
+    default: "us-west-2"
+  - name: role-name
+    description: The name of the role that needs to be created.
+  steps:
+  - name: create-vpc
```

Comment:
wrong step name. May be copy paste error ? :D

Reply:
yeah..updated